core/src/test/scala/unit/kafka/server/DynamicBrokerConfigTest.scala

```diff
@@ -100,37 +99,6 @@ class DynamicBrokerConfigTest {
     }
   }
 
-  @Test
```

Comment:
This is tested in `UncleanLeaderElectionTest`

Reply:
Which test exercises the dynamic config path?

Reply:
`testTopicUncleanLeaderElectionEnableWithAlterTopicConfigs()` starts a cluster with unclean leader election disabled and enables it via the Admin API.